### PR TITLE
DCOS-11957: Add redirect after destroying a service

### PR DESCRIPTION
**Pull request by nLight**
This PR adds a delayed redirect after a service got destroyed via the `ServiceDestroyModal`.
We have to delay the redirect because Router redirects faster than our code manages to close the modal which results in the Modal inviting you to destroy the root `/` group as well.

### CODE REVIEW

**Comment by nLight** [comment]
run unit tests

**Comment by orlandohohmeier** [comment]
L👀ks good!

**Comment by orlandohohmeier** [comment]
🥗  All test pass locally and the test failure is unrelated to the introduced changes.

File: plugins/services/src/js/components/modals/ServiceDestroyModal.js [hunk 1]
```diff
@@ -9,6 +10,11 @@ import Pod from '../../structs/Pod';
 import Service from '../../structs/Service';
 import ServiceTree from '../../structs/ServiceTree';
 
+const REDIRECT_DELAY = 500;
```
Comment (wavesoft) [inline plugins/services/src/js/components/modals/ServiceDestroyModal.js]: This feels a bit random. It's a good idea to comment what's the reasoning behind this number.

**Pull request merged**